.gitignore

```diff
@@ -1 +1 @@
-public/*
+site/*
```

Comment:
We might want to also add .gitignore support for Python development, eg. for virtualenvs. [The .gitignore for MkDocs-Material itself](https://raw.githubusercontent.com/squidfunk/mkdocs-material/master/.gitignore) might be a good starting point.

```suggestion
# Dependencies
/__pycache__
/venv
/.venv

# Build files
/site

# Caches and logs
*.log
```